Bugzilla/Constants.pm

```diff
@@ -430,9 +426,6 @@ use constant FIELD_TYPE_BUG_URLS => 7;
 use constant FIELD_TYPE_KEYWORDS => 8;
 use constant FIELD_TYPE_DATE => 9;
 use constant FIELD_TYPE_INTEGER => 10;
-use constant FIELD_TYPE_USER     => 20;
-use constant FIELD_TYPE_USERS    => 21;
-use constant FIELD_TYPE_BUG_LIST => 22;
```

Comment:
Does a placeholder need to be left behind here reminding people that 20..22 were allocated at one point, or is it safe to reuse them immediately / accidentally?